src/main/java/net/szum123321/window_title_changer/mixin/MinecraftClientMixin.java

```diff
@@ -23,14 +23,26 @@
     @Inject(method = "getWindowTitle", at = @At("HEAD"), cancellable = true)
     public void getAlternativeWindowTitle(CallbackInfoReturnable<String> ci){
         String builder = WindowTitleChanger.config.windowTitle;
+        
+        public static final MinecraftClient MC = MinecraftClient.getInstance();
```

Comment:
This is mixin. The code gets executed within Minecraft Clients class, and so, at runtime 'this' points to client instance. Please replace MinecraftClient.getInstance with `(MinecraftClient)(Object)this`. Also, why do you declare it as public static final? It's local variable.